sdks/python/apache_beam/transforms/display_test.py

```diff
@@ -22,12 +22,76 @@
 from datetime import datetime
 import unittest
 
+import hamcrest as hc
+from hamcrest.core.base_matcher import BaseMatcher
+
 import apache_beam as beam
 from apache_beam.transforms.display import HasDisplayData
 from apache_beam.transforms.display import DisplayData
 from apache_beam.transforms.display import DisplayDataItem
 
 
+def make_nspace_display_data(component):
+  dd = DisplayData.create_from(component)
+  nspace = '{}.{}'.format(component.__module__, component.__class__.__name__)
+  return nspace, dd
```

Comment:
The usage of this method for assertions feels strange because half of the output (`nspace`) feeds into the 'expected' display data, and the other half (`dd`) represents the 'actual'. It doesn't seem intuitive-- without reading the implementation I might expect the return `nspace` to be pulled from the generated display data and represent the 'actual nspace'.

Is it necessary to validate `nspace` on each transform display data test? Since this is generated by the SDK rather than from inputs I think we should only need one test in the display data unit tests and then disregard it for other validations.